packages/at_commons/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-## 4.1.3
+## 5.0.0
+- [Breaking Change]feat: Add isEncrypted flag to update verb metadata for both true and false
```

Comment:
Change to "Emit the `isEncrypted` value in the metadata if it is false"

Reply:
Done